### DISCUSSION TITLE
BRouter no street names in the turn-by-turn directions

### DISCUSSION THREAD

**Discussion by SF4User** [devemux86/cruiser#220] [Q&A] [answered]
I have now loaded the BRouter to have offline edit in the cruiser for multiple country maps.
Everything worked before, now I have this problem and I can't find a solution. I have now completely deleted BRouter and am now working with Mapsforge again. That worked fine before. . .
But my problem is still present. what is my problem does anyone know the solution?

![Screenshot Cruiser](https://github.com/devemux86/cruiser/assets/132825580/0bcfc12c-2c43-45fd-a934-2490c9c015a4)

**Comment by devemux86** [comment, marked as answer]
@SF4User 

- Mapsforge are offline maps

- BRouter is for offline routing
See also the https://github.com/devemux86/cruiser/discussions/137

Note: Brouter currently does not provide street names: https://github.com/abrensch/brouter/issues/55

**Reply by SF4User** [reply]
I have now uninstalled/deleted everything related to BRouter. . . cleaned up the Androide directory and switched back to Mapsforge.
Then all tours that the BRouter "touched" deleted and reloaded all tours new from my Kurviger Cloud. . . now everything is ok again!
Thanks for the support! devemux86 - extremely fast response wow!  👍👍

**Reply by devemux86** [reply]
Kurviger routes can be imported in Cruiser and remain unchanged.

If you need offline re-routing in navigation, you can use BRouter (offline).
It only calculates until next waypoint, the curvy routes remain unchanged.

**Reply by SF4User** [reply]
I had a running system. . . with the import of the BRouter app and the sections was not like before. . .
I like this cruiser! When it was still known as Kurviger #2, it took me 2-3 attempts before everything worked perfectly. I was very happy with that.
But now the cruiser, with these options:
- BRouter
- GraphHopper
- MapQuest
- OpenRouteService
- OSRM
- Valhalla etc. etc. etc. etc.
then the many services:
- BRouter
- BikeRouter
- Cycle Streets
- GraphHopper
- GraphHopper offline . . . etc. etc. etc.

I can't see through it. I use Cruiser and/or Kurviger as navigation for the motorcycle . . . and for nothing else.
This is all too much for a native user . . . 1000 possibilities . . .
for people. whose purpose in life is to play around with navigation apps, that may be great . . . for one user: navigation from A-B-C-D-E . . . to Target and done!  . . . it is that too much.
No criticism of your app! it is functionally great. . . but i find it too mutch overloaded.
now it works, all is OK!
![Screenshot_20230620_213830_Cruiser](https://github.com/devemux86/cruiser/assets/132825580/4790d6ff-95da-400c-86e7-6f8c18f6734f)

**Reply by devemux86** [reply]
@SF4User 

I only mentioned BRouter (offline), if you need re-rerouting in navigation without internet.

If you just want to import your Kurviger routes and navigate them,
there is nothing needed to change, import and start navigation. :slightly_smiling_face: 

---

@SF4User from your image you seem to have changed the map rendering engine?

Please use the default **VTM (OpenGL)** in "Map | Advanced" settings to have better speed, map rotation, etc.

**Reply by SF4User** [reply]
OK! 👍👍
